js/sales.js

```diff
@@ -123,6 +123,65 @@
         return dataCell;
     }
 
+    function getNewStore(event){
+        event.preventDefault();
+        const form = event.target;
+        const storeName = form.querySelector('#store-name').value;
+        const minHourlyCustomers = parseInt(form.querySelector('#min-hourly-customer').value);
+        const maxHourlyCustomers = parseInt(form.querySelector('#max-hourly-customer').value);
+        const avgCookiesPerCustomer = parseInt(form.querySelector('#avg-cookies-per-customer').value);
+        const newStore = new CookieStore(storeName, minHourlyCustomers, maxHourlyCustomers, avgCookiesPerCustomer);
+        cookieStoreLocations.push(newStore);
+        addNewStore(newStore);
+        form.querySelector('input[type=\'submit\']').disabled = true;
```

Comment:
I'm surprised this query finds an element, I always thought you had to use double-quotes for attributes in selectors. Interesting to know.